roxie/ccd/ccdmain.cpp

```diff
@@ -1397,7 +1397,8 @@ int CCD_API roxie_main(int argc, const char *argv[], const char * defaultYaml)
                         }
                         const char *soname =  roxieFarm.queryProp("@so");
                         const char *config  = roxieFarm.queryProp("@config");
-                        Owned<IHpccProtocolPlugin> protocolPlugin = ensureProtocolPlugin(*protocolCtx, soname);
+                        // NB: leaks, but cannot be Owned<> until we solve scope properly
```

Comment:
It's really "until we fix the bug in ensureProtocolPlugin whereby some paths return a linked object and others do not."

Reply:
There is a JIRA for the proper fix:  https://track.hpccsystems.com/browse/HPCC-25664

I can put together a fix, unless @mckellyln wants to look at it.